configuration/pih/htmlforms/hiv/section-hiv-state.xml

```diff
@@ -156,6 +156,243 @@
                 jq("#editChwNameDiv").hide();
             }
           });
+
+    });
+</script>
+</excludeIf>
+
+<excludeIf velocityTest="$fn.getObs($encounter, 'PIH:14535')">
+<script type="text/javascript">
+
+    jq(document).ready(function() {
```

Comment:
It might not be possible, but these three functions seem to share a lot of similar code, would be great if we could abtract that out into a utility function.

Also, would be helpful to add some comments saying what this is all doing, it's not immediately apparent to me.

Reply:
I understand, that's why I mentioned in the comment that I have something to fix. I will do it.

Reply:
No worries, thanks @louidorjp !